services/provider/server/server.go

```diff
@@ -816,9 +818,15 @@ func (s *OCSProviderServer) GetStorageClaimConfig(ctx context.Context, req *pb.S
 				&pb.ExternalResource{
 					Name: "ceph-rbd",
 					Kind: "VolumeReplicationClass",
-					Data: mustMarshal(map[string]string{
-						"replication.storage.openshift.io/replication-secret-name": provisionerSecretName,
-						"mirroringMode": "snapshot",
+					Data: mustMarshal(&replicationv1alpha1.VolumeReplicationClassSpec{
```

Comment:
Is the client operator parsing this info now instead of a map?

Reply:
Yes, the client operator is parsing based on the the kind we send